src/hardware/input/mouse_config.cpp

```diff
@@ -302,21 +353,11 @@ static void config_init(Section_prop &secprop)
 	                    "      using mirrored display mode or using an AV receiver's HDMI input for\n"
 	                    "      audio-only listening.");
 
-	prop_multi = secprop.AddMultiVal("mouse_sensitivity", only_at_start, ",");
-	prop_multi->Set_help(
+	prop_str = secprop.Add_string("mouse_sensitivity", always, "100,100");
+	prop_str->Set_help(
 	        "Mouse sensitivity for the horizontal and vertical axes, as a percentage\n"
```

Comment:
I'd say "Global mouse sensitivity" at the start, then I'd suggesting appending this at the end:

"Mouse sensitivity can be further fine-tuned per mouse interface using the internal MOUSECTL.COM tool available on the Z drive."

Reply:
OK, added more information. Also changed default to '100', as it is currently.